lib/ioki/model/driver/base.rb

```diff
@@ -7,11 +7,6 @@ class Base < Ioki::Model::Base
         def self.specification_scope
           'driver_api'
         end
-
-        attribute :id,         on: :read, type: :string
```

Comment:
Not sure about this change. I usually like explicit over implicit, but after this change `Driver::Base` deviates from the rest of the base models. The "note that this does not inherit from" comment is not the best solution either I admit. Maybe we can find something else?

Reply:
IMHO inheritance isn't used correctly when some classes don't inherit from the class they should be inheriting from, semantically speaking. This goes for the other APIs as well. If not all classes share these attributes, then we could introduce another abstract class or I'd be happy to simply add the missing :id, :type and date attributes to all classes.

Reply:
> IMHO inheritance isn't used correctly when some classes don't inherit from the class they should be inheriting from, semantically speaking

I agree.

I'm Ok with both versions (abstract class or adding fields), whatever is easier for you. But I think consistency is important, so I think it would be good to change the other models in this PR as well.

Reply:
Cool! How about we merge this and I'll do that in a different PR?

Reply:
:-) I just tried to suggested to it in **this** PR, because I'd want to avoid a state where inheritance is used differently in the different "APIs". But hey, I'm surprised I'm still being asked anyway :-). So. Sure. Go ahead :-)